README.md

```diff
@@ -18,10 +18,18 @@ Nominatim, or general analysis.
 
 ## Installing ##
 
-Most Linux distributions include osm2pgsql. It is also available on macOS with [Homebrew](http://brew.sh/).
+**Linux**: Most distributions include osm2pgsql.
 
-Unoffical builds for Windows are available from [AppVeyor](https://ci.appveyor.com/project/openstreetmap/osm2pgsql/history) but you need to find the right build artifacts. The latest
-release is [0.92.0](https://ci.appveyor.com/api/projects/openstreetmap/osm2pgsql/artifacts/osm2pgsql_Release.zip?tag=0.92.0).
+**macOS**: osm2pgsql is available with [Homebrew](http://brew.sh/).
+
+**Windows**: Unoffical builds are available from [AppVeyor](https://ci.appveyor.com/project/openstreetmap/osm2pgsql/history). The latest
+release is [0.94.0](https://ci.appveyor.com/api/projects/openstreetmap/osm2pgsql/artifacts/osm2pgsql_Release.zip?tag=0.94.0). Just unzip the archive into a directory of your choice. You may need Admnistrator permissions in order to unzip into `C:\Program Files (x86)`.
+
+_Note:_ `osm2pgsql` expects the default style file at `C:/libs/share/osm2pgsql/default.style`. In order to avoid an error message use <code>--style <em>&lt;osm2pgsql directory&gt;</em>\default.style</code> or run the following commands once:
+* `mkdir C:\libs\share`
+* <code>mklink /D C:\libs\share\osm2pgsql "<em>&lt;osm2pgsql directory&gt;</em>"</code> 
```

Comment:
I think that this is too detailed. Users who know what they are doing, don't need this instruction. All other users should just stick to adding `--style FILE` to every invocation of Osm2pgsql.

Reply:
The fact is that the use of `--style FILE` is not needed on Linux. As far as I could see, it isn't part of all the documented examples.
Even users that "know what they're doing" on Windows may be unaware of the assumptions taken by the authors of the README.

Reply:
It is need on Linux if you don't install Osm2pgsql but just build it and execute the resulting binary.

Reply:
osm2pgsql now looks in the current directory for the default.style on Windows. This should solve the issue. At least in theory. Would be nice if somebody could test this.